Tests/LeafKitTests/LeafErrorTests.swift

```diff
@@ -0,0 +1,48 @@
+/// Place all tests related to verifying that errors ARE thrown here
+
```

Comment:
General test reorganization - test cases specifically verifying that errors ARE and SHOULD be thrown